scripts/establish_conda_env.sh

```diff
@@ -232,13 +253,26 @@ do
   shift
 done
 
+# get installation manager
+INSTALL_MANAGER="$(echo $INSTALL_MANAGER | tr '[a-z]' '[A-Z]')"
+echo $INSTALL_MANAGER
+if [[ "$INSTALL_MANAGER" != "CONDA" && "$INSTALL_MANAGER" != "PIP" ]];
```

Comment:
&& => ||?

Reply:
It should be an .and. operator not an .or.   Right?

Reply:
you are right